### ISSUE TITLE
`ray` not in requirements.txt

### ISSUE THREAD

**Issue by cossatot** [gem/oq-engine#8182]
I'm working from a fresh install using latest master.  When using `run_calc` from a script I get this error:

```python
File ~/src/gem/oq-engine/openquake/baselib/parallel.py:671, in Starmap.init(cls, distribute)
    670 try:
--> 671     from ray.util.multiprocessing import Pool
    672     cls.pool = Pool(cls.num_cores, init_workers)

ModuleNotFoundError: No module named 'ray'
```
Obviously an easy fix on my machine with `pip install ray` but perhaps this should be in the requirements?

**Comment by micheles** [comment]
No, this is not right, ray is not officially supported, you should never go there. I guess the problem is that you are getting a ModuleNotFoundError instead of the expected ImportError. I will try to fix that.
